blocks/header/header.js

```diff
@@ -85,14 +85,84 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
   }
 }
 
+/**
+ * Open the sidebar on click of hamburger
+ */
+function openSidebar(block, target = '.nav-hamburger', clsName = '.nav-sidebar-new') {
+  const hamburger = block.querySelector(target);
+
+  hamburger.addEventListener('click', () => {
+    const sidebar = block.querySelector(clsName);
+    const isSidebarOpen = sidebar.classList.contains('open');
+    if (!isSidebarOpen) {
+      sidebar.classList.add('open');
+    }
+  });
+}
+
+/**
+ * Close the sidebar on click of close
+ */
+function closeSidebar(block, clsName = '.nav-sidebar-new') {
+  const sidebar = block.querySelector(clsName);
+  const closeBtn = block.querySelector(`header nav ${clsName} .icon.icon-close`);
+
+  closeBtn.addEventListener('click', () => {
+    const isSidebarOpen = sidebar.classList.contains('open');
+    if (isSidebarOpen) {
+      sidebar.classList.remove('open');
+      document.body.style = '';
+    }
+  });
+}
+
+function closeLanguageNavOnClickOutside(globe) {
+  document.addEventListener('click', (event) => {
+    const target = event.target.closest('li.active');
+    if (target !== globe || !target) {
+      globe.classList.remove('active');
+    }
+  });
+}
+
+function toggleLanguageNav(block) {
+  const languageToggleEls = block.querySelectorAll('.nav-sidebar-new > ul li:last-child, .nav-sections > ul li:last-child');
+  const ACTIVE = 'active';
+
+  [...languageToggleEls].forEach((languageToggleEl) => {
+    languageToggleEl.addEventListener('click', () => {
+      const isActive = languageToggleEl.classList.contains(ACTIVE);
+      if (isActive) {
+        languageToggleEl.classList.remove(ACTIVE);
+      } else {
+        languageToggleEl.classList.add(ACTIVE);
+      }
+    });
+    closeLanguageNavOnClickOutside(languageToggleEl);
+  });
+}
+
+function decorateSecondarySidebar(block) {
+  const sidebarUlEl = block.querySelector('.nav-sections ul li:nth-child(3) ul');
+  sidebarUlEl?.classList.add('nav-sidebar-phone');
+
+  openSidebar(block, '.nav-sections .icon.icon-phone', '.nav-sidebar-phone');
+  closeSidebar(block, '.nav-sidebar-phone');
+}
+
 /**
  * decorates the header, mainly the nav
  * @param {Element} block The header block element
  */
 export default async function decorate(block) {
+  window.onscroll = function () {
```

Comment:
I think her we could use 
```*.js
    window.addEventListener('scroll', () => {
      /* code */
    });
```

Reply:
Of course. Are you going to change it? @thomsebastin